packages/docs/src/pages/en/blog/building-a-basic-nuxt-application-with-vuetify.md

```diff
@@ -0,0 +1,565 @@
+
+Let's verify the proper functioning of this mechanism by displaying the products.vue page. We assume that each page of the application has been created similarly to the index.vue page. These pages will be created in the next section if needed.
+
+![](https://cdn.vuetifyjs.com/docs/images/blog/building-a-basic-nuxt-application-with-vuetify/12.png){ height=500 }
+
+We can now navigate from page to page within the application.
+
+## Creating the other pages of the application
+
+The products.vue page, displayed previously, must first be created. We create it using the same principle as the index.vue page that was created earlier.
+
+Here is the code describing each of the components associated with the pages of the application:
+
+```html { resource="pages/products.vue" }
+<template>
+  <v-container class="fill-height d-flex flex-column align-center justify-center text-center">
+    <v-icon size="64">mdi-cube-outline</v-icon>
+
+    <div class="mt-3">MyApplication's Products Page</div>
+  </v-container>
+</template>
+```
+
+## Adding buttons to the toolbar
+
+Finally, we might want to insert new buttons into the toolbar at the top of the screen, on the right side. These buttons can provide direct access to one of the application’s pages, without using the previously implemented naviga- tion menu.
+
+We therefore insert buttons in the form of icons into the application's tool- bar. The app.vue component becomes:
+
+```html
+<template>
+  <v-app>
+    <!-- Toolbar at the top of the screen -->
+    <v-app-bar app color="primary" height="48">
+      <!-- Title of the application with adjusted margin and font size -->
+      <v-toolbar-title class="text-h6 ml-3">
+      <v-icon @click="drawer = !drawer">mdi-apps</v-icon>
+      <span class="ml-1"> My Application </span>
+      </v-toolbar-title>
+
+      <!-- Spacer to push the following elements to the right -->
+      <v-spacer />
+
+      <!-- Menu icons on the right side of the toolbar -->
+      <v-btn icon @click="navigateTo('/')">
+        <v-icon>mdi-home</v-icon>
+      </v-btn>
+      <v-btn icon @click="navigateTo('/products')">
+        <v-icon>mdi-cube-outline</v-icon>
+      </v-btn>
+      <v-btn icon @click="navigateTo('/contact')">
+        <v-icon>mdi-email</v-icon>
+      </v-btn>
+    </v-app-bar>
+    <!-- Navigation drawer for menu actions -->
+    <v-navigation-drawer v-model="drawer">
+      <v-list>
+      <!-- Menu items in the drawer with icons -->
+      <v-list-item @click="navigateTo('/')">
+        <v-list-item-title>
+          <v-icon left>mdi-home</v-icon>
+          Home
+        </v-list-item-title>
+      </v-list-item>
+
+      <v-list-item @click="navigateTo('/products')">
+        <v-list-item-title>
+          <v-icon left>mdi-cube-outline</v-icon>
```

Comment:
`<v-icon left>mdi-cube-outline</v-icon>` to `<v-icon icon="mdi-cube-outline" />`